src/app/pages/result/result.component.ts

```diff
@@ -61,6 +68,14 @@ export class ResultComponent implements OnInit, OnDestroy {
     return Math.round((num + Number.EPSILON) * 100) / 100;
   }
 
+  updateScoring(difference: number): void {
+    this.result.scoring = this.roundTwoDecimals(difference <= 0 ? this.result.scoring + difference : this.result.scoring - difference);
```

Comment:
Schönes Ding! 👍 Wenn du die Berechnung direkt in den Benford Service ausgelagert hast, reicht es die Methode direkt in der ngOnInit aufzurufen 😄